docs/sources/alerting/monitor/meta-monitoring/_index.md

```diff
@@ -67,19 +83,31 @@ This metric is a histogram that shows you the number of seconds taken to send no
 
 > These metrics are not available at present in Grafana Cloud.
 
-## Grafana Mimir
+## Metrics for Grafana Mimir
+
+{{% admonition type="note" %}}
+This metric is available in OSS, on-premise, and Grafana Cloud.
+{{% /admonition %}}
 
-Meta monitoring in Grafana Mimir requires having a Prometheus/Mimir server, or other metrics database, collecting and storing metrics exported by the Mimir ruler.
+To meta monitor in Alertmanager, you need a Prometheus/Mimir server, or another metrics database to collect and store metrics exported by the Mimir ruler.
```

Comment:
```suggestion
To meta monitor Grafana Mimir-managed alerts, open source and on-premise users need a Prometheus/Mimir server, or another metrics database to collect and store metrics exported by the Mimir ruler.
```

Reply:
Add something like 

Grafana Cloud customers can find this metric in their `grafanacloud-usage` datasource.

Reply:
note that its going to have a slightly different name though so that might cause confusion. In Grafana Cloud its:
`grafanacloud_instance_rule_evaluation_failures_total:rate5m{}`